Resources/doc/reference/child_admin.rst

```diff
@@ -0,0 +1,129 @@
+Create child admins
+-------------------
+
+Let us say you have a ``PlaylistAdmin`` and a ``VideoAdmin``. You can optionally declare the ``VideoAdmin``
+to be a child of the ``PlaylistAdmin``. This will create new routes like, for example, ``/playlist/{id}/video/list``,
+where the videos will automatically be filtered by post.
```

Comment:
Please comply with the sf docs standards:

> [Each line should break approximately after the first word that crosses the 72nd character (so most lines end up being 72-78 characters);](http://symfony.com/doc/current/contributing/documentation/standards.html)

Reply:
There are more offending lines below.

Reply:
I did not write a single character in this text!

Reply:
Should I edit it anyway?

Reply:
Oh right I forgot! Drop it then.